src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__big"
+      data-qa="big"
+    >
+      <label class="search--form__label search--form__label_big">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="search--form__input search--form__input_big"
```

Comment:
Again, the class names `search--form__label_big` and `search--form__input_big` should be named semantically, representing the content rather than the style or size.